app/src/main/java/com/example/tmdb/ui/MainActivity.kt

```diff
@@ -23,6 +24,7 @@ class MainActivity : ComponentActivity() {
     @Inject
     lateinit var navigator: Navigator
 
+    @OptIn(ExperimentalMaterial3Api::class)
```

Comment:
You can move it to class scope with annotation `@ExperimentalMaterial3Api`. Just make it consistent with other experimental annotations

Reply:
Fixed in new commit